ecms_base/modules/custom/ecms_api_publisher/src/EcmsApiPublisherInstall.php

```diff
@@ -118,6 +118,7 @@ private function createSimpleOauthConsumer(EntityInterface $user): ?EntityInterf
 
     $values = [
       'user_id' => $user->id(),
+      'client_id' => $this->t('eCMS Publisher'),
```

Comment:
Just confirming that the client_id is a label and not a machine name?

Reply:
The client_id is used as a label in the list of consumers page. The UUID is still used to uniquely identify a consumer. Some developers also add the UUID to the client_id but that is not as easy to read in the UI. 

The existing code had to identify the consumer with UUID, so I set the client_id like a label.

Reply:
Do you think that it would be better for the client_id to match the UUID? After all the UUID is set from a local object with a method getClientId()

Reply:
I think this is fine if it is just used in the admin. The name `client_id` had me curious of it's intent.